extension/lib/settings_wrapper.js

```diff
@@ -96,6 +110,18 @@ class Settings {
 		});
 	}
 
+	// Set the URLs that the user has visited
+	setVisitedUrls(visitedUrls) {
+		chrome.storage.local.set({[settingKey.VISITED_URLS]: visitedUrls}, function() {
+			if (chrome.runtime.lastError) {
+				console.log("Failed to save speed setting: " + chrome.runtime.lastError);
```

Comment:
save "visited urls"

[Future idea, no need to reply]
-- Hrmm I'm starting to think that I'll refactor to 

setSetting(settingKey, val)
getSetting(settingKey)

and leave the comments in the setting key defns.
These codes all look the same lol

Re:storage limit, this can be handled by that one fcn too in a systematic manner for all settings. E.g. each setting has a byte size limit.

Reply:
To clarify -- the console.log string has copied pasted string